### PR TITLE
fix: [PIE-5667]: restrict new service creation in templates for org level

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

- Restrict new service creation in templates for org level

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 #### 1 Issues 
* Code issue 1 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11702)

File: src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/DeployServiceEntitySpecifications.tsx [hunk 1]
```diff
@@ -232,7 +237,7 @@ export default function DeployServiceEntitySpecifications({
               scope === Scope.PROJECT
                 ? allowableTypes
                 : ((allowableTypes as MultiTypeInputType[]).filter(
-                    item => item !== MultiTypeInputType.FIXED
+                    item => item !== MultiTypeInputType.FIXED && item !== MultiTypeInputType.EXPRESSION
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/DeployServiceEntitySpecifications.tsx:240]: Expression support has been added by BE. Behind FF however, we can enable this

Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/DeployServiceEntitySpecifications.tsx:240]:
```suggestion
                    item => item !== MultiTypeInputType.FIXED
```

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Approved

**Pull request merged**